README.md

```diff
@@ -19,32 +19,35 @@ The repository has three goals:
+2. The second package, `ros2_control_demo_description`, stored URDF-description files, rviz configurations and meshes for the demo robots.
+3. The third package, `ros2_control_demo_hardware`, implements the hardware interfaces described in the roadmap.
 The examples simulate a simple *RRbot* internally to provide sufficient test and demonstration data and reduce external dependencies.
 This package does not have any dependencies except `ros2` core packages and can, therefore, be used on SoC-hardware of headless systems.
-2. The second package, `ros2_control_demo_hardware_gazebo`, uses a Gazebo simulator to simulate the *RRBot* and its physics.
-This package is useful to test the connection of `ros2_control` to the Gazebo simulator and to detect any missing plugins.
-3. The third package `ros2_control_demo_robot` holds examples for *RRbot* URDF-description, launch files and controllers.
 
 This repository demonstrates the following `ros2_control` concepts:
 
 * Creating of `*HardwareInterface` for a System, Sensor, and Actuator.
-* Creating a robot description in the form of URDF files
-* Loading the configuration and starting a robot using launch files
-* Control of two joints of *RRBot*
-* Using simulated robots and starting `ros_control` with Gazebo simulator
-* Implementing of controller switching strategy for a robot
-* Using joint limits and transmission concepts in `ros2_control`
+* Creating a robot description in the form of URDF files.
+* Loading the configuration and starting a robot using launch files.
+* Control of two joints of *RRBot*.
+* Using simulated robots and starting `ros2_control` with Gazebo simulator.
+* Implementing of controller switching strategy for a robot.
+* Using joint limits and transmission concepts in `ros2_control`.
 
 ## Quick Hints
 
 These are some quick hints, especially for those coming from a ROS1 control background:
 
-* There are now three categories of hardware interface: *Sensor*, *Actuator*, and *System*. Sensor is for individual sensors; Actuator is for individual actuators; System is for any combination of multiple sensors/actuators. You could think of a Sensor as read-only.
-* ros(1)_control only allowed three hardware interface types: position, velocity, and effort. ros2_control allows you to create any interface type by defining a custom string. For example, you might define a `position_in_degrees` or a `temperature` interface. The most common (position, velocity, acceleration, effort) are already defined as constants in hardware_interface/types/hardware_interface_type_values.hpp.
-* In ros2_control, all parameters for the driver are specified in the URDF. The ros2_control framework uses the <ros2_control> tag in the URDF.
-* <ros2_control> tags in the URDF must be compatible with the controller's configuration.
-* PLUGINLIB_EXPORT_CLASS macro is required when implementing an interface.
+* There are now three categories of hardware interface: *Sensor*, *Actuator*, and *System*.
```

Comment:
```suggestion
* There are now three categories of hardware interface components: *Sensor*, *Actuator*, and *System*.
```
Shouldn't this be hardware interface components? Later we talk of the different hardware interfaces types that can be defined, and it may be a bit confusing.

Reply:
I'd leave out the word 'interfaces' completely. We've been using `components` everywhere else in the code base.